Jetcaster/core/src/main/java/com/example/jetcaster/core/player/EpisodePlayer.kt

```diff
@@ -85,4 +85,9 @@ interface EpisodePlayer {
      * Rewinds a currently played episode by a given time interval specified in [duration].
      */
     fun rewindBy(duration: Duration)
+
+    /**
+     * Changes the speed of which the player increments
+     */
+    fun changePlaySpeed(duration: Duration)
```

Comment:
Total nit, this should be `setPlayerSpeed` and probably could just be a public var

Reply:
Definitely much better. Fixed in fbe75e28